src/sidebars/favourites/hover-service.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.hover', [])
+        .factory('hoverService', [function() {
+            var self = this;
+            var hoverTargets = [];
+
+            function get() {
+                return hoverTargets;
+            }
+
+            function add(hoverArea, dropTarget, stockCode) {
+                var newHoverTarget = { hoverArea: hoverArea, dropTarget: dropTarget, code: stockCode };
+                if (hoverTargets.indexOf(newHoverTarget) === -1) {
+                    // This target is new
+                    hoverTargets.push(newHoverTarget);
+                }
+            }
+
+            return {
+                get: get,
+                add: add
+            };
+        }]);
```

Comment:
I have a feeling that as favourites are removed they'll mistakenly persist in the service's local array. They should be removed.. somehow.

As far as the user is concerned I don't think this will have an implication as they extra targets will be ignored, but still it's messy.